packages/core-mobile/app/vmModule/ApprovalController/ApprovalController.ts

```diff
@@ -64,6 +65,16 @@ class ApprovalController implements VmModuleApprovalController {
 
             break
           }
+          case RpcMethod.BITCOIN_SIGN_TRANSACTION: {
```

Comment:
so this wasn't working before then

Reply:
Embarrassingly, no. I couldn’t actually test the btc bridging because we only had enough Bitcoin in the mainnet.